docs/variables.md

```diff
@@ -4,108 +4,106 @@ linkTitle: "Variable Substitutions"
 
 | Variable | Description |
 | -------- | ----------- |
-| `params.<param name>` | The value of the param at runtime. |
+| `params.<param name>` | The value of the parameter at runtime. |
 | `resources.inputs.<resource name>.path` | The path to the input resource's directory. |
 | `resources.outputs.<resource name>.path` | The path to the output resource's directory. |
-| `results.<result name>.path` | The path to the file where a Task's result should be written. |
-| `workspaces.<workspace name>.path` | The path to the mounted workspace. |
-| `workspaces.<workspace name>.claim` | The name of the PersistentVolumeClaim used as a volume source for the workspace or empty string if a volume source other than PersistentVolumeClaim was used. |
-| `workspaces.<workspace name>.volume` | The name of the volume populating the workspace. |
-| `credentials.path` | The path to the location of credentials written by the `creds-init` init container. |
+| `results.<result name>.path` | The path to the file that stores the `Task's` result. |
+| `workspaces.<workspace name>.path` | The path to the mounted `Workspace`. |
+| `workspaces.<workspace name>.claim` | The name of the `PersistentVolumeClaim` specified as a volume source for the `Workspace`. Empty string for other volume types. |
+| `workspaces.<workspace name>.volume` | The name of the volume populating the `Workspace`. |
+| `credentials.path` | The path to the credentials written by the `creds-init` init container. |
 
-### PipelineResource Variables
+### `PipelineResource` variables available in a `Task`
 
-Each PipelineResource exposes its own set of variables. Below the variables are grouped by
-PipelineResource type. These are available in `Tasks`.
+Each supported type of `PipelineResource` specified within a `Task` exposes a unique set
+of variables. This section lists the variables exposed by each type. You can access a
+variable via `resources.inputs.<resourceName>.<variableName>` or
+`resources.outputs.<resourceName>.<variableName>`.
 
-Each variable is accessible via `resources.inputs.<resource name>.<variable name>` or
-`resources.outputs.<resource name>.<variable name>`.
-
-#### Git PipelineResource
+#### Variables for the `Git` type
 
 | Variable | Description |
 | -------- | ----------- |
-| `name` | The resource's name |
-| `type` | `"git"` |
-| `url` | The URL to the Git repo |
-| `revision` | The revision to be checked out. |
-| `depth` | The integer value of the resource's `depth` param. |
-| `sslVerify` | The value of the resource's `sslVerify` param: `"true"` or `"false"`. |
-| `httpProxy` | The value of the resource's `httpProxy` param. |
-| `httpsProxy` | The value of the resource's `httpsProxy` param. |
-| `noProxy` | The value of the resource's `noProxy` param. |
-
-#### PullRequest PipelineResource
+| `name` | The name of the resource. |
+| `type` | Type value of `"git"`. |
+| `url` | The URL of the Git repository. |
+| `revision` | The revision to check out. |
```

Comment:
There's another one to add here which was only recently introduced in 0.12: `refspec`.  Description would be something like "The value of the resource's `refspec` parameter."

Reply:
Added.